services/red/src/ambassador/artiso.AdsdHotel.Red.Ambassador/Program.cs

```diff
@@ -1,23 +1,51 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Threading.Tasks;
-using artiso.AdsdHotel.Red.Api;
+using artiso.AdsdHotel.Red.Contracts;
 using Grpc.Net.Client;
+using Date = artiso.AdsdHotel.Red.Contracts.Date;
 
 namespace artiso.AdsdHotel.Red.Ambassador
 {
     public class Program
     {
-        public static async Task Main()
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE0060:Remove unused parameter")]
+        // ReSharper disable once UnusedParameter.Local
+        static async Task Main(string[] args)
```

Comment:
For a runner example, create a `Consumer` project referencing the `Ambassador` and run the base workflow.
See `blue`, `yellow`, or `purple`.